tests/utils/utils.go

```diff
@@ -1035,3 +1147,30 @@ func ParseTeleporterMessage(unsignedMessage avalancheWarp.UnsignedMessage) *tele
 
 	return teleporterMessage
 }
+
+func GetWarpEnabledChainConfig(offChainMessages []avalancheWarp.UnsignedMessage) string {
```

Comment:
`GetChainConfigWithOffChainMessages` would be more descriptive. Warp is enabled on all chains post-Durango